README.md

```diff
@@ -8,14 +8,14 @@
 
 # Enduro
 
-Enduro is a tool designed to automate the processing of transfers in multiple
-Archivematica pipelines. It's part of a preservation solution that is being
-used by the [National Health Archive (NHA)] and the [National Center for Truth
-and Reconciliation (NCTR)].
+Enduro is a new application under development by Artefactual Systems. Originally
+created as a more stable replacement for Archivematica’s automation tools
+library of scripts, it has since evolved into a flexible tool to be paired with
+preservation applications to provide initial ingest activities such as content
+and structure validation, packaging, and more.
 
-It's a **proof of concept** at its very early stages. It aims to cover our
-client's needs while exploring new and innovative ways to build durable and
-fault-tolerant workflows suited for preservation.
+Enduro aims to cover our client's needs while exploring new and innovative ways
```

Comment:
nitpick - should perhaps be `clients'` needs, to make plural clients possessive? Or, `our client needs` (drop the s entirely)?

Reply:
Thanks! That's legacy text, so may have been from the time when there was literally only one client :joy: